### ISSUE TITLE
IPv6 interface has no IP in ReaR enviroment

### ISSUE THREAD

**Issue by damm620** [rear/rear#3273]
* ReaR version ("/usr/sbin/rear -V"):
2.7

* OS version ("cat /etc/os-release" or "lsb_release -a" or "cat /etc/rear/os.conf"):
CentOS 9

* ReaR configuration files ("cat /etc/rear/site.conf" and/or "cat /etc/rear/local.conf"):

/etc/rear/site.conf
```
OUTPUT=ISO               # Creates an ISO file - other e.g. RAMDISK
BACKUP=TSM               # Backup/Restore method - other e.g. rsync
COPY_AS_IS_TSM=( "/etc/adsm/ /opt/tivoli/tsm/client" )  # Files which should be in the ISO
# Skip language files to reduce the ISO size
COPY_AS_IS_EXCLUDE_TSM=( "/opt/tivoli/tsm/client/ba/bin/ZH_CN/*" "/opt/tivoli/tsm/client/ba/bin/ZH_TW/*" "/opt/tivoli/tsm/client/ba/bin/IT_IT/*" "/opt/tivoli/tsm/client/ba/bin/CS_CZ/*" "/opt/tivoli/tsm/client/ba/bin/ES_ES*" "/opt/tivoli/tsm/client/ba/bin/FR_FR/*" "/opt/tivoli/tsm/client/ba/bin/HU_HU/*" "/opt/tivoli/tsm/client/ba/bin/KO_KR/*" "/opt/tivoli/tsm/client/ba/bin/PL_PL/*" "/opt/tivoli/tsm/client/ba/bin/PT_BR/*" "/opt/tivoli/tsm/client/ba/bin/JA_JP/*" "/opt/tivoli/tsm/client/ba/bin/RU_RU/*" "/opt/tivoli/tsm/client/ba/bin/plugins/*" "/opt/tivoli/tsm/client/api/bin64/CS_CZ*" "/opt/tivoli/tsm/client/api/bin64/DE_DE/*" "/opt/tivoli/tsm/client/api/bin64/dsmenc" "/opt/tivoli/tsm/client/api/bin64/dsm.opt.smp" "/opt/tivoli/tsm/client/api/bin64/dsm.sys.smp" "/opt/tivoli/tsm/client/api/bin64/dsmtca" "/opt/tivoli/tsm/client/api/bin64/EN_US" "/opt/tivoli/tsm/client/api/bin64/ES_ES" "/opt/tivoli/tsm/client/api/bin64/FR_FR" "/opt/tivoli/tsm/client/api/bin64/HU_HU" "/opt/tivoli/tsm/client/api/bin64/IT_IT" "/opt/tivoli/tsm/client/api/bin64/JA_JP" "/opt/tivoli/tsm/client/api/bin64/KO_KR" "/opt/tivoli/tsm/client/api/bin64/PL_PL" "/opt/tivoli/tsm/client/api/bin64/PT_BR" "/opt/tivoli/tsm/client/api/bin64/RU_RU" "/opt/tivoli/tsm/client/api/bin64/sample" "/opt/tivoli/tsm/client/api/bin64/ZH_CN" "/opt/tivoli/tsm/client/api/bin64/ZH_TW" )
ONLY_INCLUDE_VG=("vg00")
CHECK_CONFIG_FILES=( '/etc/adsm/' '/etc/drbd/' '/etc/drbd.conf' '/etc/lvm/lvm.conf' '/etc/multipath.conf' '/etc/rear/' '/etc/udev/udev.conf' '/opt/tivoli/tsm/client/ba/bin/dsmc' )
PROGS_TSM=(dsmc)
PROGS=( ${PROGS[@]} "vconfig vi vim" )
NON_FATAL_BINARIES_WITH_MISSING_LIBRARY=("/opt/tivoli/tsm/client/*|/usr/local/ibm/gsk8_64/bin/")
GRUB_RESCUE=y
AUTOEXCLUDE_MULTIPATH=y  # Default Wert=y - multipath devices are not created/saved. Which is usually not necessary
                         # If these must be restored nevertheless e.g. "rm -rf /". The restore must be performed manually afterwards
TSM_RESULT_SAVE=n        # Don't save ISO in TSM
KEEP_BUILD_DIR=n         # Remove build directory (dir in /tmp) in any case to prevent full filesystem
USER_INPUT_TIMEOUT=900   # Increase timeout for user input from 5 to 15 minutes
```

/etc/rear/local.conf
```
ONLY_INCLUDE_VG=("evg00")
```

* Hardware vendor/product (PC or PowerNV BareMetal or ARM) or VM (KVM guest or PowerVM LPAR):
VM (VMware)

* System architecture (x86 compatible or PPC64/PPC64LE or what exact ARM device):
x86

* Firmware (BIOS or UEFI or Open Firmware) and bootloader (GRUB or ELILO or Petitboot):
BIOS, Grub

* Storage (local disk or SSD) and/or SAN (FC or iSCSI or FCoE) and/or multipath (DM or NVMe):
local SSD

* Storage Layout (only normal lsblk)
```
NAME                                          MAJ:MIN RM  SIZE RO TYPE MOUNTPOINTS
sda                                             8:0    0   80G  0 disk
├─sda1                                          8:1    0    1G  0 part /boot
└─sda2                                          8:2    0   79G  0 part
  ├─vg00-root                                 253:0    0    2G  0 lvm  /
  ├─vg00-swap                                 253:1    0    4G  0 lvm  [SWAP]
  ├─vg00-usr                                  253:2    0    4G  0 lvm  /usr
...
sr0                                            11:0    1 1024M  0 rom
```

* Description of the issue (ideally so that others can reproduce it):
the running system have the following network config:
```
ip -br a
lo               UNKNOWN        127.0.0.1/8 ::1/128
eth0             UP             10.XX.XX.50/27 fe80::XXX:XXXX:fe01:9a79/64
eth1             UP             2a01:XXX:XXXX:XXXX::55/64 fe80::XXX:XXXX:fe01:9a7a/64
```
so IPv6 on the eth1. 

mkrescue runs normally:
```
TMPDIR=/tmp /usr/sbin/rear mkrescue; echo $?
Failed to copy all contents of /lib/modules/5.14.0-474.el9.x86_64 (dangling symlinks could be a reason)
/usr/lib64/systemd/libsystemd-core-252.so requires additional libraries
        libsystemd-shared-252.so => not found
ReaR recovery system in '/tmp/rear.vO06Zi3IVKjl8Hf/rootfs' needs additional libraries, check /var/log/rear/rear-SERVER.log for details
0
```

Then after booting into the ReaR the interface eth1 has no IP. This is a big issue because eth1 is our backup interface :(

I do not have any protocol of what the "ip a" output looks exactly, but I just want to open this issue to ask if the bug is already known and there is already a known fix for this?

**Comment by gdha** [comment]
@damm620 PR #2907 fixed this and will be part of ReaR v3.0 once it gets released.
When will it be released? The answer you can find in https://relax-and-recover.org/events/ - "Why is there no new Release? Nobody pays for the basics :-("

**Comment by jsmeix** [comment]
I think this issue can be closed because it is fixed via
https://github.com/rear/rear/pull/2907
